pkg/reconciler/workload/namespace/namespace_reconcile.go

```diff
@@ -64,13 +65,13 @@ var (
 
 func init() {
 	// This matches namespaces that haven't been scheduled yet
-	if req, err := labels.NewRequirement(ClusterLabel, selection.DoesNotExist, []string{}); err != nil {
+	if req, err := labels.NewRequirement(ScheduledClusterLabel, selection.DoesNotExist, []string{}); err != nil {
```

Comment:
does this mean we need a label to tell that we are scheduled, and then one per cluster? Would it make sense to have a list of cluster names in this one, and then call the other "state"?